src/resolvers/myndlaResolvers.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
```

Comment:
Veit det berre er kopiering av kode, men fila er ny.
```suggestion
 * Copyright (c) 2023-present, NDLA.
```